requirements.txt

```diff
@@ -35,5 +35,6 @@ martor @ git+https://github.com/DMOJ/martor.git
 netaddr
 webauthn<1
 bleach
```

Comment:
You probably don't need this line anymore.

Reply:
Deleted.